docs/modules/operation/pages/provisioning/import-handler.adoc

```diff
@@ -42,117 +40,158 @@ Examples:
+[source]
+----
+dns://my-dns-server/myzone.com/portland/?expression=^por-.*
+----
 
-If your expression requires URL encoding (for example you need to use a `?` in the expression) it must be properly encoded.
+This URL will import all nodes from the same server and zone, but will only import the nodes in the zone matching the regular expression `^port-.*`.
+The matching nodes will be assigned to a requisition named 'portland'.
+If your expression requires URL encoding, for example you need to use a `?` in the expression, it must be properly encoded.
 
- dns://my-dns-server/myzone.com/portland/?expression=^por[0-9]%3F
+[source]
+----
+dns://my-dns-server/myzone.com/portland/?expression=^por[0-9]%3F
+----
 
 == Location
 
-You can use the `location` parameter to assign a single location to all nodes in this requisition:
+You can use the `location` parameter to assign a single location to all nodes in this requisition.
+This would only apply if you are using Minions in your network.
 
-  dns://my-dns-server/myzone.com/portland/?location=Raleigh
+[source]
+----
+dns://my-dns-server/myzone.com/portland/?location=Raleigh
+----
 
 By prefixing with a `~`, you can specify a url-encoded regular expression to determine the location based on the host entries.
 The following example derives the location based on the subdomain.
-It uses a regular expression `^(?:.*\.|)(.*?)\.hs-fulda\.de\.$` with exactly one capturing group containing the subdomain:
+This example uses a URL-encoded regular expression `^(?:.*\.|)(.*?)\.hs-fulda\.de\.$` with exactly one capturing group containing the subdomain.
 
-  dns://my-dns-server/hs-fulda.de/?location=~%5E%28%3F%3A.%2A%5C.%7C%29%28.%2A%3F%29%5C.hs-fulda%5C.de%5C.%24
+[source]
+----
+dns://my-dns-server/hs-fulda.de/?location=~%5E%28%3F%3A.%2A%5C.%7C%29%28.%2A%3F%29%5C.hs-fulda%5C.de%5C.%24
+----
 
 This will associate nodes for the host entries `foo.e46.hs-fulda.de.` and `bar.e46.hs-fulda.de.` to the location `e46` while hosts like `aaa.g51.hs-fulda.de` and `bbb.g51.hs-fulda.de` will be assigned location `g51`.
 
-TIP: You can use online tools like `https://www.urlencoder.org` to encode your parameters.
+TIP: You can use online tools like https://www.urlencoder.org to encode your parameters.
 
 === DNS setup
 
-Currently, the DNS server requires to be setup to allow a zone transfer from the {page-component-title} server.
-It is recommended that a secondary DNS server is running on {page-component-title} and that the {page-component-title} server be allowed to request a zone transfer.
+The DNS server must be configured to allow a zone transfer from the {page-component-title} server.
+// Question: Is this still recommended to setup a local DNS secondary?
```

Comment:
```
// Question: Is this still recommended to setup a local DNS secondary?
```
I am not sure. Will have to ask someone who might know.

Reply:
We can remove this comment / TODO here. You have to cover security issues when you allow zone transfers between your OpenNMS and the DNS server. For us, as OpenNMS, it's first a matter of fact we need to access the DNS server using AXFR/IXFR. IMHO, what strategies you need to address this topic of security in your DNS environment is a different story.

Reply:
```suggestion
```